test/fixtures/stitch_mongo_fixture.js

```diff
@@ -34,7 +34,9 @@ export default class StitchMongoFixture {
     this.stitch = new StitchService;
   }
 
-  async setup() {
+  async setup(options) {
```

Comment:
This might be out of scope, but the use of this function now seems kind of odd. I'd expect that having createApp be true would create a new empty app, not import one over. Thoughts on this?

Reply:
I think it's out of scope for this ticket but I definitely agree; I filed a ticket (STITCH-779) to refactor the test fixtures a bit. I would like to get rid of test_app_config.json entirely and rip out the code for starting up the test server, since we don't use it at all.